* replace asm/segment.h (obsolete) by asm/uaccess.h

taken from upstream cvs (required for ppc compliation)

diff -Nur lirc-0.8.0.orig/drivers/lirc_it87/lirc_it87.c lirc-0.8.0/drivers/lirc_it87/lirc_it87.c
--- lirc-0.8.0.orig/drivers/lirc_it87/lirc_it87.c	Tue Oct  4 23:10:04 2005
+++ lirc-0.8.0/drivers/lirc_it87/lirc_it87.c	Sat Mar 18 17:03:19 2006
@@ -58,7 +58,7 @@
 #include <linux/delay.h>
 #include <linux/poll.h>
 #include <asm/system.h>
-#include <asm/segment.h>
+#include <asm/uaccess.h>
 #include <asm/io.h>
 #include <asm/irq.h>
 #include <asm/fcntl.h>
diff -Nur lirc-0.8.0.orig/drivers/lirc_serial/lirc_serial.c lirc-0.8.0/drivers/lirc_serial/lirc_serial.c
--- lirc-0.8.0.orig/drivers/lirc_serial/lirc_serial.c	Sat Dec 17 15:56:57 2005
+++ lirc-0.8.0/drivers/lirc_serial/lirc_serial.c	Sat Mar 18 17:03:29 2006
@@ -91,7 +91,7 @@
 #include <linux/poll.h>
 
 #include <asm/system.h>
-#include <asm/segment.h>
+#include <asm/uaccess.h>
 #include <asm/io.h>
 #include <asm/irq.h>
 #include <asm/fcntl.h>
diff -Nur lirc-0.8.0.orig/drivers/lirc_sir/lirc_sir.c lirc-0.8.0/drivers/lirc_sir/lirc_sir.c
--- lirc-0.8.0.orig/drivers/lirc_sir/lirc_sir.c	Sat Jan 14 15:57:24 2006
+++ lirc-0.8.0/drivers/lirc_sir/lirc_sir.c	Sat Mar 18 17:03:35 2006
@@ -75,7 +75,7 @@
 #include <linux/delay.h>
 #include <linux/poll.h>
 #include <asm/system.h>
-#include <asm/segment.h>
+#include <asm/uaccess.h>
 #include <asm/io.h>
 #include <asm/irq.h>
 #include <asm/fcntl.h>
